cordova/plugins/InAppBrowser.d.ts

```diff
@@ -152,9 +152,9 @@ interface InAppBrowser extends Window {
      *                  loaderror: event fires when the InAppBrowser encounters an error when loading a URL.
      *                  exit: event fires when the InAppBrowser window is closed.
      * @param callback  the function that executes when the event fires. The function is
-     *                  passed an InAppBrowserEvent object as a parameter.
+     *                  passed an Event object as a parameter.
      */
-    removeEventListener(type: "exit", callback: (event: InAppBrowserEvent) => void): void;
+    removeEventListener(type: "exit", callback: (event: Event) => void): void;
```

Comment:
Were these necessary? If you're entirely sure that `"exit"` events will give you back an `InAppBrowserEvent`, then those should stay the same.

The only ones you *can't* be sure about is the non-specialized signature (i.e. the one whose `type` parameter doesn't have a string literal type)`. So those should take `Event`.